soh/soh/Enhancements/controls/GameControlEditor.cpp

```diff
@@ -313,8 +313,8 @@ namespace GameControlEditor {
             UIWidgets::Spacer(5);
             SohImGui::BeginGroupPanel("Walk Modifier", ImGui::GetContentRegionAvail());
             UIWidgets::PaddedEnhancementCheckbox("Toggle modifier instead of holding", "gWalkSpeedToggle", true, false);
-            UIWidgets::EnhancementSliderFloat("Modifier 1: %d %%", "##WalkMod1", "gWalkModifierOne", 0.0f, 5.0f, "", 1.0f, true);
-            UIWidgets::EnhancementSliderFloat("Modifier 2: %d %%", "##WalkMod2", "gWalkModifierTwo", 0.0f, 5.0f, "", 1.0f, true);
+            UIWidgets::PaddedEnhancementSliderFloat("Modifier 1: %d %%", "##WalkMod1", "gWalkModifierOne", 0.0f, 5.0f, "", 1.0f, true, true, false, true);
+            UIWidgets::PaddedEnhancementSliderFloat("Modifier 2: %d %%", "##WalkMod2", "gWalkModifierTwo", 0.0f, 5.0f, "", 1.0f, true, true, false, true);
```

Comment:
could we get a before/after shot of the padding here?

Reply:
Before:  
![image](https://user-images.githubusercontent.com/30329717/222877952-054a0eac-ab95-4eb4-b204-dde0c8eae042.png)

Reply:
After:
![image](https://user-images.githubusercontent.com/30329717/222878004-e5ce302b-20bb-4bdb-8f67-1b31a2f37fd5.png)

Reply:
Specifically, I added the space after the slider, so you'll see that there's room between the "Modifier 2" text and the slider above it, as well as room between the bottom slider and the divider line below it.